changelog.d/9401-non-utf8-argv.md

```
@@ -0,0 +1,61 @@
+### Fixed
+
+- **A non-UTF-8 byte in `argv` no longer aborts the process.**
+  `claude -p $'\xff\xfe\x80abc\xc3\x28'` died with **SIGABRT** and a raw Rust
+  backtrace —
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)